lib/rules/nesting-depth.js

```diff
@@ -2,6 +2,15 @@
 
 var helpers = require('../helpers');
 
+var lodash = require('lodash');
+
+var compiledCSSDoesNotCreateDepth = function (selector) {
+  return lodash.isArray(selector.content) &&
```

Comment:
We don't have this package available, we were using certain modules of loadash rather than including the whole library.